actions/action-releaser/contrib/semver

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
```

Comment:
This is third-party open-source code?  Make sure to follow the license requirements and include the license if needed.

Reply:
Yeah I'm not sure where it is originally from, I saw that another tag bumping action in this repo uses it so I just copied it.